### DISCUSSION TITLE
【灌水】关于这个所谓的第一律者

### DISCUSSION THREAD

**Discussion by OttoApocalypse01** [share121/inter-knot#3179] [其他]
![3fa7b146626d4db5bd102aa559e9dbc2](https://github.com/user-attachments/assets/35beedc5-ba82-4855-bf59-97316e5f552d)
只有我一个人对瓦尔特无感吗，真不明白为什么这么多人这么喜欢他......

**Comment by Vellichor35** [comment]
是哪一代啊<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >
